common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -29,7 +29,8 @@ class LibraryRoot(XBlock):
     advanced_modules = List(
         display_name=_("Advanced Module List"),
         help=_("Enter the names of the advanced components to use in your library."),
-        scope=Scope.settings
+        scope=Scope.settings,
+        xml_node=True,
```

Comment:
@cahrens See the [comment here from @bradenmacdonald](https://github.com/edx/edx-platform/pull/6846#issuecomment-72307600)

This is allows the field to be serialized in a manner where it can be unserialized. It looks like if a List or Dict field is not given this token, it will be spat out as a unicode representation of its value, which works fine for strings and integers, but not so much for lists and dictionaries.